app/views/flood_risk_engine/enrollments/steps/_local_authority.html.erb

```diff
@@ -1 +1,14 @@
-local_authority
+<fieldset>
+
+  <legend class="visuallyhidden"><%= t('.legend') %></legend>
+
+  <%= form_group_and_validation(form, :name) do %>
+      <div class="form-group">
+        <label class="form-label" for="authority-name"><%= t('.label') %></label>
+        <p class="form-hint"><%= t('.para_text') %></p>
```

Comment:
Nice. Shouldn't we use the more Railsy '<%= f.label_for .. %>'  to guarantee labels are always associated with the right input eg `<%= f.label :name, t('.label'), class: 'form-label' %>`?